aimax-osm/src/main/java/aimax/osm/routing/OsmActionsFunction.java

```diff
@@ -20,8 +20,8 @@
  */
 public class OsmActionsFunction implements ActionsFunction {
 
-	public static enum OneWayMode {IGNORE, TRAVEL_FORWARD, TRAVEL_BACKWARDS}; 
-	

```

Comment:
The Java language designers made it optional to put a semicolon at the end of the enum declaration (when no other declarations for this enum follow):